Lib/test/test_pathlib.py

```diff
@@ -246,6 +136,46 @@ class P(_BasePurePathSubclass, self.cls):
         for parent in p.parents:
             self.assertTrue(parent.init_called)
 
+    def _check_drive_root_parts(self, arg, expected):
+        def f(parts):
+            path = self.cls(*parts)
+            return path.drive, path.root, list(path.parts)
```

Comment:
Since path.parts are tuples, they could be compared directly to tuple `expected`s instead of converted to lists to match incorrect `expected`s.  Similarly, tuple `arg`s iterate as well as list `arg`s.  Since the tuples would become compiled constants instead of list-creation code, a change would make the tests run a bit faster.  Besides the search-replace fixup effort, I suppose the counter argument is that the lists are easier to read.  To counter that, I would insert a space or comma in the ']))' replacement to avoid ending with ')))'.

Reply:
Done. I changed the signature of `_check_drive_root_parts` so that the expected values don't need to be packed into a 3-tuple, which eliminates one of the trailing parentheses. I think it's reasonably readable, but I could still add a comma or a space if you think that's better?